app/views/admin/borrowings/_form_search.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= search_form_for [:admin, @q], class: "form-inline" do |f| %>
+  <div class="form-group">
+    <span><%= t ".borrow_code" %></span>
+    <%= f.search_field :borrow_code_cont,
+      class: "form-control code_value",
+      placeholder: t(".borrow_code") %>
+  </div>
+  <div class="form-group">
+    <span><%= t ".user_name" %></span>
+    <%= f.search_field :user_name_cont,
```

Comment:
cont là gì

Reply:
cont là 1 option của ransack và use truy vấn Like trong SQL
`SELECT COUNT(*) FROM `borrowings` LEFT OUTER JOIN `users` ON `users`.`deleted_at` IS NULL AND `users`.`id` = `borrowings`.`user_id` WHERE `borrowings`.`deleted_at` IS NULL AND `users`.`name` LIKE '%test%'`

Reply:
sao lại là select count(*) nhỉ

Reply:
?